borgmatic/borg/create.py

```diff
@@ -351,7 +351,9 @@ def create_archive(
     sources = deduplicate_directories(
         map_directories_to_devices(
             expand_directories(
-                tuple(location_config.get('source_directories', ())) + borgmatic_source_directories
+                tuple(location_config.get('source_directories', ()))
+                + borgmatic_source_directories
+                + tuple(global_arguments.used_config_paths)
```

Comment:
Does this need a test? Maybe something along the lines of the existing `test_create_archive_with_patterns_calls_borg_with_patterns_including_converted_source_directories()`?